CommsRadioSoundSwitcher.cs

```diff
@@ -91,36 +91,36 @@ public void Awake()
             }
             else
             {
-                Debug.LogError("CommsRadioSoundSwitcher: couldn't get properties from siblings");
+                Main.DebugLog(() => "CommsRadioSoundSwitcher: couldn't get properties from siblings");
```

Comment:
These are fine to use `Debug` and not be controllable IMO, since they indicate a bug in the mod and shouldn't fire in the usual case.